src/contributors/MapContributor.ts

```diff
@@ -216,6 +243,22 @@ export class MapContributor extends Contributor {
                     this.aggregationField = collection.params.centroid_path;
                 }
             });
+
+        if (this.includeFeaturesFields) {
+            this.allIncludedFeatures = new Set();
+            this.includeFeaturesFields.forEach(f => this.allIncludedFeatures.add(f));
+            this.includeFeaturesFieldsSet = new Set(this.includeFeaturesFields);
+            if (this.normalizationFields) {
+                this.normalizationFields.forEach(f => {
```

Comment:
Could you use filter instead of for each ?

Reply:
done